Packages/Blockchain/Sources/Ethereum/EthereumService.swift

```diff
@@ -287,3 +306,14 @@ extension EthereumService: ChainIDFetchable {
             .map(as: JSONRPCResponse<BigIntable>.self).result.value.description
     }
 }
+
+
+// MARK: - ChainLatestBlockFetchable
+
+extension EthereumService: ChainLatestBlockFetchable {
+    public func getLatestBlock() async throws -> String? {
```

Comment:
```suggestion
    public func getLatestBlock() async throws -> String {
```

String should be non optional